environment-nohoomd.yml

```diff
@@ -1,22 +1,17 @@
 name: uli-init
 channels:
+  - jaimergp/label/unsupported-cudatoolkit-shim
```

Comment:
I think for this env (gpu) it is OK to pull in cudatoolkit

Reply:
I can test this out. I don't really differentiate between `environment-nohoomd.yml` and `environment.yml` as one being GPU and one being CPU. Rather, the no-hoomd yml file is for use when you're using an already compiled and set up hoomd (like in a container) and don't want it also being installed via anaconda.  In that case, if it was compiled to run on a gpu then I assume the cudatool kit package is already installed.

Reply:
Ok, I tested it out for both scenarios. Using the `-nohoomd.yml` worked with my locally compiled hoomd, so not pulling cuda via anaconda didn't disrupt anything. Also, using the regular environment file  (with hoomd) still pulled hoomd-gpu.